### PR TITLE
add intMax

### PR DESCRIPTION
**Pull request by Tatenda-D-Mawoneke**

### CODE REVIEW

File: Warmup-1/intMax/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export function intMax(a: number, b: number, c: number): number {
+  //   return Math.max(a, b, c);
```
Comment (JustinGuedes) [inline Warmup-1/intMax/src/index.ts:2]: Why is this commented out?

File: Warmup-1/intMax/test/index.test.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+describe("intMax", () => {
+  it("should return the largest value 3 when passed 1,2,3", () => {
+    const result = intMax(1, 2, 3);
+    assert.equal(result, 3);
+  });
+  it("should return the largest value 3 when passed 1,3,2", () => {
+    const result = intMax(1, 3, 2);
+    assert.equal(result, 3);
+  });
+  it("should return the largest value 3 when passed 3,2,1", () => {
```
Comment (JustinGuedes) [inline Warmup-1/intMax/test/index.test.ts:19]: This seems like an incorrect test name?

**Review by JustinGuedes — CHANGES_REQUESTED** [review]

File: Warmup-1/intMax/src/index.ts (hunk 1 above)
Comment (Tatenda-D-Mawoneke) [reply Warmup-1/intMax/src/index.ts:2]: It is a solution, but I commented it out for the sake of practicing loops.

File: Warmup-1/intMax/test/index.test.ts (hunk 2 above)
Comment (Tatenda-D-Mawoneke) [reply Warmup-1/intMax/test/index.test.ts:19]: I don't see anything wrong with the test name, but i do see a defect in the test data and I have corrected the test data.

Comment (JustinGuedes) [reply Warmup-1/intMax/test/index.test.ts:19]: The test name dictates what the test does, so if the test does something else then either the test name or test needs to change, so your comment is invalid when arguing something was wrong with the test name

Comment (JustinGuedes) [reply Warmup-1/intMax/test/index.test.ts:19]: Also is this PR ready for review? This conversation is not resolved

File: Warmup-1/intMax/src/index.ts (hunk 1 above)
Comment (JustinGuedes) [reply Warmup-1/intMax/src/index.ts:2]: Fair, but in the future rather create a commit with it and then commit on top of it with a different solution. That way it's in history. Comments are not necessarily good practice

**Pull request merged**